embed_gen.go

```diff
@@ -3,5 +3,5 @@ package gotemplate
 
 import "embed"
 
-//go:embed _template _template/.azure-pipelines.yml _template/.dockerignore _template/.editorconfig _template/.githooks _template/.github _template/.gitignore _template/.gitlab-ci.yml _template/.golangci.yml
+//go:embed _template _template/.azure-pipelines.yml _template/.dockerignore _template/.editorconfig _template/.githooks _template/.github _template/.gitignore _template/.gitlab-ci.yml _template/.golangci.yml _template/.vscode
```

Comment:
Could you remove the .vscode folder here again pls?. I think it landed there accidentally on make generate right?
We should probably exclude stuff that is in gitignore during generation, I will note that 👀